src/filesystem.cpp

```diff
@@ -21,6 +21,15 @@
 
 bool assure_dir_exist( const std::string &path )
 {
+    // TEMPORARY until we drop VS2019 support
```

Comment:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87846 says GCC was behaving the same way until the fix in GCC 8.3. Our documentation says minimum supported GCC is 8.1, so a bump in GCC support policy is also needed to remove this workaround in the future.

Reply:
That's a doc mistake.  It's supposed to be minimum gcc 9.1 now since dropping Code::Blocks.  I'll PR a fix to the docs.